dart-sdk/rust/Cross.toml

```diff
@@ -0,0 +1,35 @@
+[build]
+xargo = false
+
+[build.env]
+volumes = [ "SDK_DIR=../", ]
+
+[target.aarch64-linux-android]
+image = "ghcr.io/cross-rs/aarch64-linux-android:main"
+
+[target.x86_64-linux-android]
+image = "ghcr.io/cross-rs/x86_64-linux-android:main"
+
+[target.armv7-linux-androideabi]
+image = "ghcr.io/cross-rs/armv7-linux-androideabi:main"
```

Comment:
x86 32-bit target needed here?

Reply:
these are actually the default images so this config is probably unnecessary. I'm debading whether to keep this or remove, might remove for simplicity